platform/pulp/app/models/storage.py

```diff
@@ -1,6 +1,7 @@
 import os
 import errno
-import shutil
+import io
+import stat
```

Comment:
I don't mind ordering alphabetically but, fyi, alphabetizing imports is not part of pep8.
```
 Imports should be grouped in the following order:

    standard library imports
    related third party imports
    local application/library specific imports
```